source/features/embed-gist-via-iframe.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	const embedMenuItem = select('.select-menu-item')!;
+	select('.select-menu-button')!.textContent = 'Embed via <script>';
+	select('.select-menu-item-heading', embedMenuItem)!.innerHTML = 'Embed via <code>&lt;script&gt;</code>';
+	select('.description', embedMenuItem)!.innerHTML = 'Embed this gist in your website via <code>&lt;script&gt;</code>.';
+
+	const iframe = `<iframe src="https://gist.github.com${location.pathname}.pibb"></iframe>`;
+	embedMenuItem.insertAdjacentElement(
```

Comment:
This can be replaced by `.after()`. See https://github.com/sindresorhus/refined-github/pull/760

Reply:
Oh thanks I'm rebuilding this via `cloneNode`